src/main/osd/osd_elements.c

```diff
@@ -578,13 +579,28 @@ static void osdElementEscTemperature(osdElementParms_t *element)
     }
 }
 
+#endif // USE_ESC_SENSOR
+
+#if defined(USE_ESC_SENSOR) || defined(USE_RPM_FILTER)
+int getEscRpm(int i)
```

Comment:
Please move this up into the section with all the utility/formatting functions and out of the section with the element drawing functions.

Reply:
this is fixed in the latest push